docs/core/deploying/native-aot/warnings/il3055.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "IL3055: P/invoke method declares a parameter with an abstract delegate"
+description: "Correctness of interop for abstract delegates cannot be guaranteed after native compilation - the marshalling code for the delegate might not be available"
+ms.date: 09/01/2022
+ms.topic: reference
+ms.author: michals
+author: MichalStrehovsky
+f1_keywords:
+  - "IL3055"
+---
+# IL3055: P/invoke method declares a parameter with an abstract delegate
+
+## Cause
+
+P/invoke marshalling code needs to be generated ahead of time. If marshalling code for a delegate wasn't pregenerated, p/invoke marshalling will fail with an exception at runtime.
```

Comment:
```suggestion
P/Invoke marshalling code needs to be generated ahead of time. If marshalling code for a delegate wasn't pregenerated, P/Invoke marshalling will fail with an exception at run time.
```

Reply:
Looks like we always capitalize P/Invoke: https://docs.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke